lib/models/base-locale.js

```diff
@@ -465,15 +465,20 @@ async function updateHabilitation(baseLocaleId, habilitationId) {
   )
 }
 
-async function removeSoftDeletedBALs() {
+async function removeSoftDeletedBALsOlderThanOneYear() {
   const now = new Date()
   const deleteTime = subMonths(now, 12)
 
-  const softDeletedBALs = await mongo.db.collection('bases_locales').deleteMany({
+  const softDeletedBALs = await mongo.db.collection('bases_locales').find({
     _deleted: {$lt: deleteTime}
-  })
+  }).toArray()
+
+  await Promise.all(softDeletedBALs.map(async bal => {
```

Comment:
Un for..of séquentiel plutôt qu'une exécution en parallèle est plus adapté pour ce genre de traitement.
Sinon ça créé énormément de requêtes qui vont polluer le serveur MongoDB.

Par ailleurs il manque des `await` devant le deleteOne et le cleanContent.